src/kOS/Communication/ConnectivityManager.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace kOS.Communication
+{
+    public interface ConnectivityManager
+    {
+        double GetDelay(Vessel vessel1, Vessel vessel2);
+    }
+}
```

Comment:
Oh yeah... huh.  I guess it's just used for inter-vessel communications, that doesn't have to go to Safe.  So long as we can try to move inter-processor communications into Safe eventually (doesn't have to happen before we merge this, unless it's easy enough to do).